index.html

```diff
@@ -554,6 +554,775 @@ <h2>OpenID Connect</h2>
+        }
+      }, {
+        "id": "did:example:123456789abcdefghi#keys-1",
+        "type": "Multikey", <span class="comment">// external (property value)</span>
+        "controller": "did:example:pqrstuvwxyz0987654321",
+        "publicKeyMultibase": "z6MkmM42vxfqZQsv4ehtTjFFxQ4sQKS2w6WR7emozFAn5cxu"
+      }],
+      <span class="comment">...</span>
+    }
+            </pre>
+          </section>
+
+          <section>
+            <h3>Multikey</h3>
+            <p>
+The Multikey data model is a specific type of <a>verification method</a> that
+utilizes the [[MULTICODEC]] specification to encode key types into a single
+binary stream that is then encoded using the [[MULTIBASE]] specification.
+            </p>
+
+            <p>
+When specifing a `Multikey`, the object takes the following form:
+            </p>
+
+            <dl>
+              <dt>type</dt>
+              <dd>
+The value of the `type` property MUST contain the string `Multikey`.
+              </dd>
+              <dt><dfn class="lint-ignore">publicKeyMultibase</dfn></dt>
+              <dd>
+The `publicKeyMultibase` property is OPTIONAL. If present, its value MUST be a
+[[MULTIBASE]] encoded [[MULTICODEC]] value.
+              </dd>
+              <dt><dfn class="lint-ignore">secretKeyMultibase</dfn></dt>
+              <dd>
+The `secretKeyMultibase` property is OPTIONAL. If present, its value MUST be a
+[[MULTIBASE]] encoded [[MULTICODEC]] value.
+              </dd>
+            </dl>
+
+            <p>
+An example of a Multikey is provided below:
+            </p>
+
+            <pre class="example nohighlight"
+              title="Multikey encoding of a Ed25519 public key">
+{
+  "@context": ["https://w3id.org/security/multikey/v1"],
+  "id": "did:example:123456789abcdefghi#keys-1",
+  "type": "Multikey",
+  "controller": "did:example:123456789abcdefghi",
+  "publicKeyMultibase": "z6MkmM42vxfqZQsv4ehtTjFFxQ4sQKS2w6WR7emozFAn5cxu"
+}
+            </pre>
+
+            <p>
+In the example above, the `publicKeyMultibase` value starts with the letter `z`,
+which is the [[?MULTIBASE]] header that conveys that the binary data is
+base58-encoded using the Bitcoin base-encoding alphabet. The decoded binary data
+[[?MULTICODEC]] header is `0xed`, which specifies that the remaining data
+is a 32-byte raw Ed25519 <a>public key</a>.
+            </p>
+
+            <p>
+The Multikey data model is also capable of encoding secret keys, whose subtypes
+include <em>symmetric keys</em> and <em>private keys</em>.
+            </p>
+
+            <pre class="example nohighlight"
+              title="Multikey encoding of a Ed25519 secret key">
+{
+  "@context": ["https://w3id.org/security/suites/secrets/v1"],
+  "id": "did:example:123456789abcdefghi#keys-1",
+  "type": "Multikey",
+  "controller": "did:example:123456789abcdefghi",
+  "secretKeyMultibase": "z3u2fprgdREFtGakrHr6zLyTeTEZtivDnYCPZmcSt16EYCER"
```

Comment:
Especially problematic given there is no way to validate the shape of the bytes here, according to a standard that defines what they are for each key type.